src/app/(mobile-ui)/layout.tsx

```diff
@@ -1,24 +1,23 @@
 'use client'
 
-import '../../styles/globals.bruddle.css'
 import { Button, NavIcons, NavIconsName } from '@/components/0_Bruddle'
-import Link from 'next/link'
-import { useEffect, useState, useMemo } from 'react'
-import Modal from '@/components/Global/Modal'
-import { useWallet } from '@/context/walletContext'
-import { useZeroDev } from '@/context/walletContext/zeroDevContext.context'
-import { usePathname } from 'next/navigation'
-import classNames from 'classnames'
+import CloudsBackground from '@/components/0_Bruddle/CloudsBackground'
+import { useToast } from '@/components/0_Bruddle/Toast'
 import Icon from '@/components/Global/Icon'
-import { useRouter } from 'next/navigation'
-import WalletToggleButton from '@/components/Home/WalletToggleButton'
-import { useAuth } from '@/context/authContext'
+import Modal from '@/components/Global/Modal'
 import HomeWaitlist from '@/components/Home/HomeWaitlist'
+import WalletToggleButton from '@/components/Home/WalletToggleButton'
 import { peanutWalletIsInPreview } from '@/constants'
-import CloudsBackground from '@/components/0_Bruddle/CloudsBackground'
+import { useAuth } from '@/context/authContext'
+import { useWallet } from '@/context/walletContext'
+import { useZeroDev } from '@/context/walletContext/zeroDevContext.context'
 import { colorMap } from '@/utils'
-import { useWeb3Modal } from '@web3modal/wagmi/react'
-import { useToast } from '@/components/0_Bruddle/Toast'
+import { useAppKit } from '@reown/appkit/react'
+import classNames from 'classnames'
+import Link from 'next/link'
+import { usePathname, useRouter } from 'next/navigation'
+import { useEffect, useMemo, useState } from 'react'
+import '../../styles/globals.bruddle.css'
```

Comment:
thought: hmm I'm not sure I like having tailwind.config.js then also tailwind bruddle, then global.css then globals.bruddle.css

Seems a bit messy. Could use some refactoring